data-loader/core/src/main/java/com/scalar/db/dataloader/core/dataexport/producer/CsvProducerTask.java

```diff
@@ -144,6 +145,18 @@ private String convertToString(Result result, String columnName, DataType dataTy
       case TEXT:
         value = result.getText(columnName);
         break;
+      case DATE:
+        value = Objects.requireNonNull(result.getDate(columnName)).toString();
```

Comment:
Since we already have the null check with `result.isNull`, I don' think we need to use `Objects.requireNonNull()` here. What do you think?

Reply:
@brfrn169 san,
I added the `Objects.requireNonNull()`  to fix a warning I saw  when  adding toString conversion (Method invocation 'toString' may produce 'NullPointerException' ) on IntelliJ. I have removed it.
Thank you.

Reply:
> I added the Objects.requireNonNull() to fix a warning I saw when adding toString conversion (Method invocation 'toString' may produce 'NullPointerException' ) on IntelliJ.

@inv-jishnu Okay, in that case, you can add `assert` as follows:
```java
        LocalDate date = result.getDate(columnName);
        assert date != null;
        value = date.toString();
```

Reply:
@brfrn169 san,
Thank you.
I have updated the code as suggested.